dandi/tests/test_upload.py

```diff
@@ -12,14 +12,13 @@
 DANDIFILES_DIR = Path(__file__).with_name("data") / "dandifiles"
 
 
-def test_upload(local_docker_compose, monkeypatch, tmp_path):
+def test_upload(local_docker_compose_env, monkeypatch, tmp_path):
     DIRNAME1 = "sub-anm369963"
     FILENAME1 = "sub-anm369963_ses-20170228.nwb"
     DIRNAME2 = "sub-anm372793"
     FILENAME2 = "sub-anm372793_ses-20170508.nwb"
 
-    monkeypatch.setenv("DANDI_API_KEY", local_docker_compose["api_key"])
-    dandi_instance = "local-docker-tests"
+    dandi_instance = local_docker_compose_env["instance_id"]
```

Comment:
here and in another test below (more code would need tune up for the rename):

```suggestion
    dandi_instance_id = local_docker_compose_env["instance_id"]
```
for consistency?

Reply:
Done.